src/name_bazaar/ui/subs.cljs

```diff
@@ -61,14 +63,38 @@
      (path-for route (merge
                       (when my-address
                         {:user/address (str my-address)})
-                      params)))
+                      (if (:user/ens-name params)
+                        (set/rename-keys params {:user/ens-name :user/address})
+                        params))))
     "#" "")))
 
 (reg-sub-raw
  :resolved-route-params
  (fn [db p]
    (reaction
     (let [route-params @(subscribe [:district0x/route-params])]
-      (update
-       route-params
-       :user/address (partial try-resolving-address @db))))))
+      (resolve-params @db route-params)))))
+
+(reg-sub-raw
+ :my-resolved-address
```

Comment:
Should be called `:active-address-resolved`. + too much repetition with `:my-full-resolved-address`. Should be probably as 1 sub with passed parameter about truncation